Project.toml

```diff
@@ -0,0 +1,17 @@
+name = "ConicBenchmarkUtilities"
```

Comment:
Last time I checked project.toml files and manifest.toml files are not needed for registered packages and can confuse the package manager (https://github.com/JuliaRegistries/General/issues/22). Is there more recent guidance?

Reply:
It would seem that I do not know enough about the new package manager to understand how to use it without toml files.  For example, how does one run `test` inside the Pkg REPL?  If there is a tutorial on how to do this I can look into dropping the toml files.

For what it is worth, when setting up the toml files I use the same UUID as the General registry.  So I don't think there will be any confusion.